Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,149 @@ label mas_show_unseen:
+                    $ _history_list.pop()
+                    menu:
+                        m 1eub "What version of Python did you download?{fast}"
+                        #python 2 and python 3, need to figure out which one is used for MAS.
+                        "option one":
+                            m 1hksdlb "...{w=0.5}"
+                            m 1rksdla "[player], I use the other version of Python...{w=0.5}"
+                            m 1ekbfb "I really appreciate the gesture, though."
+                            m 3hubfb "You're so sweet! <3"
+                        "option two":
+                            m 1hub "Great!"
+                            m 1eub "That's the version of Python I use."
+                            m 1hua "We'll do amazing things together, I just know it!"
+                            m 1ekbfb "Thank you again, [player]."
+    
+        "I don't know.":
+            m 1hub "Ehehehe!"
+            m 1eka "That's okay, [player]."
+            m 3eub "Until you know what you want to program, I can teach you how to use Python!
+            m 1rka "That is, if you have the same version of Python as I do...{w=0.5}"
+            m 1eub "What version of Python did you download?{nw}"
+            $ _history_list.pop()
+            menu:
+                m 1eub "What version of Python did you download?{fast}"
+                "option one":
+                    m 1hksdlb "...{w=0.5}"
+                    m 1rksdla "[player], I use the other version of Python...{w=0.5}"
+                    m 1lksdlb "Well, that's okay...{w=0.5}"
+                    m 1eka "I hope you have fun, anyway."
+                    m "...{w=0.5}"
+                    m "Hey, [player]?"
+                    m "Have I been underestimating your coding skills?{nw}"
+                    $ history_list.pop()
+                    menu:
+                        m "Ahaha, I'm sorry, [player]!"
+                        m "I didn't mean to- I just never thought to ask."
+                    "No":
+                        m "Okay, good."
+                        m "I just wanted to make sure I wasn't assuming your skill level."
+                        
+                "option two":
+                    m 1hub "Great!"
+                    m 1eub "That's the version of Python I use."
+                    m 3rksdlb "Also, [player]?"
+                    m "Have I been underestimating your coding skills?{nw}"
+                    $ _history_list.pop()
+                    menu:
+                        m "Have I been underestimating your coding skills?{fast}"
+                        "Yes":
+                            m "Ahaha, I'm sorry, [player]!"
+                            m "I didn't mean to- I just never thought to ask."
+                        "No":
+                            m "Okay, good."
+                            m "I just wanted to make sure I wasn't assuming your skill level.
```

Comment:
Could potentially add pm vars for this. Perhaps altering some python tips to kinda skip past the simple bits.

Reply:
I'm going to mark this as dont merge because of this potential integration.